public/js/icinga/behavior/modal.js

```diff
@@ -170,7 +170,7 @@
     Modal.prototype.onKeyDown = function(event) {
         var _this = event.data.self;
 
-        if (! event.isDefaultPrevented() && event.key === 'Escape') {
+        if (_this.modalOpener && ! event.isDefaultPrevented() && event.key === 'Escape') {
```

Comment:
Nah, the modal opener isn't the right indicator. It's searching for the modal below already and passes an empty set to `hide` if there's none. Just check if the set is empty and don't cal hide then.